src/index.html

```diff
@@ -21,7 +21,360 @@
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header">
+      <a href="#home">
+        <img
+          src="./images/logo.png"
+          alt="moyo logo"
+          class="header__logo"
+        />
+      </a>
+      <nav>
+        <ul class="nav">
+          <li>
+            <a
+              href="#apple"
+              class="nav__link nav__link--is--active is-active"
+            >
+              apple
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#samsung"
+              class="nav__link"
+            >
+              samsung
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#smartphones"
+              class="nav__link"
+            >
+              smartphones
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#laptops"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              laptops & computers
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#gadgets"
+              class="nav__link"
+            >
+              gadgets
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#tablets"
+              class="nav__link"
+            >
+              tablets
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#photo"
+              class="nav__link"
+            >
+              photo
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#video"
+              class="nav__link"
+            >
+              video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog">
+      <article
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__image"></div>
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__code">Product code: 195434</p>
+
+        <section class="card__rating">
```

Comment:
The stars rating system should be made more accessible. Consider using aria-label or another method to indicate the rating to screen readers.